src/components/FormTracker/index.js

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { number, string } from 'prop-types';
 import firebase from '../../firebase';
 
 export const FormTracker = ({ age, email, id, name }) => {
+  const [edit, setEdit] = useState(true);
+  const [updateName, setUpdateName] = useState();
+
   const handleClick = () => {
```

Comment:
I would change this to `handleDeleteClick` as it's more descriptive to what the function is doing.

Reply:
yeah good point - will update!